.circleci/config.yml

```diff
@@ -14,11 +14,21 @@ jobs:
             virtualenv -p python3 /usr/local/share/virtualenvs/tap-dynamodb
             source /usr/local/share/virtualenvs/tap-dynamodb/bin/activate
             pip install .[dev]
+            pip install pytest-cov
       - run:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-dynamodb/bin/activate
             make lint
+      - run:
```

Comment:
Please update the config to current standards as requested in PR #35

Reply:
updated the config

Reply:
> > The build is failing because we need to add a parameter 'mongodb.expression' from the stitch side.
> 
> I believe once the tap-tester image is updated this will not be an issue for you.

@kspeer825 We have updated the tap-tester image as you suggested. However, the CircleCI still throws the same error as before.